src/server/Routes/Auth.js

```diff
@@ -0,0 +1,46 @@
+const express = require('express');
+const router = express.Router();
+
+// Route for registering a user
+router.post("/register", async (req, res) => {
+    const { name, email, password } = req.body;
```

Comment:
What if req does not have the name, email and password ? Shouldn't we have checks here ?

Reply:
Apply whereever applicable throughout PR.